source/NVDAObjects/IAccessible/ia2Web.py

```diff
@@ -68,6 +68,16 @@ def _get_states(self):
 			states.discard(controlTypes.STATE_EDITABLE)
 		return states
 
+	def getFocusInfo(self):

```

Comment:
I can't see where this method is actually called? perhaps it is not needed anymore?